docs/en/reference/information_schema/tables_config.md

```diff
@@ -0,0 +1,24 @@
+---
+displayed_sidebar: "English"
+---
+
+# tables_config
+
+`tables_config` provides information about the configuration of tables.
+
+The following fields are provided in `tables_config`:
+
+| **Field**        | **Description**                                              |
+| ---------------- | ------------------------------------------------------------ |
+| TABLE_SCHEMA     | Name of the database that stores the table.                  |
+| TABLE_NAME       | Name of the table.                                           |
+| TABLE_ENGINE     | Engine type of the table.                                    |
+| TABLE_MODEL      | Data model of the table. Valid values: "DUP_KEYS", "AGG_KEYS", "UNQ_KEYS" or "PRI_KEYS". |
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/30f98ca7-cc8a-44b4-9a80-5223a9bb2eb9)
Wrap the values in code format.